tfjs-backend-webgpu/src/unary_op_util.ts

```diff
@@ -168,6 +169,24 @@ const SINH = `
   let e2x = exp(a);
   return (e2x - 1.0 / e2x) / 2.0;
 `;
+const SOFTPLUS = `
+  let epsilon = 1.1920928955078125e-7;
+  let threshold = log(epsilon) + 2.0;
+
+  let too_large = a > -threshold;
+  let too_small = a < threshold;
+  let exp_a = exp(a);
+
+  if (too_large){
+    return a;
+  }
+  else if (too_small){
+    return exp_a;
+  }
+  else{
+    return log(exp_a + 1.0);
+  }
```

Comment:
We should follow some coding convention here: 1) a space between condition and { 2) else (if) is in the same line of }